packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -277,10 +277,10 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
   private _onRenderMenu(menuProps: IContextualMenuProps): JSX.Element {
     return (
       <ContextualMenu
-        className={ css('ms-BaseButton-menuHost') }
-        isBeakVisible={ true }
-        directionalHint={ DirectionalHint.bottomLeftEdge }
-        items={ menuProps.items }
+        {...menuProps}
+        className={ css('ms-BaseButton-menuhost', menuProps.className) }
+        isBeakVisible={ typeof menuProps.isBeakVisible === 'undefined' ? true : menuProps.isBeakVisible }
```

Comment:
That's how we've done this in other situations.  defaults/overridable -> menuProps -> buttonProps/set in stone